Automattic/wp-calypso
# Notices Redux: isPersistent and displayOnNextPage options

This PR continues work from #1496.

## What for?

It makes notices non permanent by default

It adds 2 properties to notices options that were present in previous notices implementation
- `isPersistent`, which makes the notice appear after reload
- `displayOnNextPage`, which displays notice on next page display

## Testing

- Go to `/me/notifications`
- Change notification settings for a site
- See that success notice disappears on route change
